README.md

```diff
@@ -137,7 +150,7 @@ twurl accounts
 Contributors
 ------------
 
-Marcel Molina <marcel@twitter.com> / @noradio  
```

Comment:
@andypiper This is for a line break, FYI (two spaces)

Reply:
I don't quite see the change here and the linter seems ok with this line, but I did remove the email address since it is no longer valid anyway.